src/API/Site/Controllers/MerchantCenter/ContactInformationController.php

```diff
@@ -243,13 +248,16 @@ protected function get_contact_information_response( ?AccountBusinessInformation
 			}
 		}
 
+		$has_address_errors = $this->settings->has_address_errors( $wc_address );
```

Comment:
💅  WDYT of `$wc_address_errors` instead of `$has_address_errors` 

`$has_address_errors` implies that it has errors.

Reply:
I think  $gla_address_errors or $mc_address_errors instead of $wc_address_errors would be better as the issue is more related to Google Merchant Center/ GLA plugin than with WC. WC is not really doing that validation. WDYT?

Reply:
That would work as well! 👌  My main concern was the prefix 'has' or 'is' which implies a boolean condition.